arrows/vxl/tests/test_image.cxx

```diff
@@ -178,7 +178,13 @@ TYPED_TEST(image_io, type)
   kwiver::vital::image_of<pix_t> img( 200, 300, TypeParam::depth );
   populate_vital_image<pix_t>( img );
 
-  auto const image_path = kwiver::testing::temp_file_name( "test-", ".tiff" );
+  std::string ext = ".tiff";
```

Comment:
This could be greatly simplified:

```
auto const image_ext = ( img.depth() == 4 ? ".png" : ".tiff" );
auto const image_path = kwiver::testing::temp_file_name( "test-", image_ext );
```

...which avoids a non-const local and gratuitous construction of a `std::string`.